waspc/ChangeLog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.14.1
```

Comment:
Wasn't sure how the versioning system worked, but looking at it now would this go in 0.13.3 instead?

Reply:
Since we will be releasing 0.14.0 in a mid-July, I believe this will get release within that version. So, write the update under the `0.14.0`.